.github/workflows/ci.yml

```diff
@@ -228,38 +228,3 @@ jobs:
           name: html-report--attempt-${{ github.run_attempt }}
           path: playwright/playwright-report
           retention-days: 14
-
-  e2e-cypress:
```

Comment:
I can see one reference to Cypress that should be remove in `.github/actions/restore-cache/action.yml` for ` ~/.cache/Cypress`.

Reply:
Ooh, good catch!